neon_transformers/text_transformers.py

```diff
@@ -78,7 +78,8 @@ def __init__(self, bus, config=None):
         self.has_loaded = False
         self.bus = bus
         self.config = self.config_core.get("utterance_transformers") or \
-            {"neon_utterance_translator": {}}
+            {"neon_utterance_translator_plugin": {},
```

Comment:
this should be done at the  default config file level, using default values here will force these plugins in ovos-core also